jsk_topic_tools/scripts/filtered_relay.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from importlib import import_module
+
+import roslib.message
+import rospy
+
+
+def expr_eval(expr, modules):
+    def eval_fn(topic, m, t):
+        return eval(expr, modules, {'m': m})
+    return eval_fn
+
```

Comment:
Would this function be divided as a module?
https://github.com/jsk-ros-pkg/jsk_common/search?q=expr_eval

Reply:
That's a good direction.
I created a PR to divide it.
https://github.com/jsk-ros-pkg/jsk_common/pull/1760
Please check it.